src/main/java/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -900,7 +900,7 @@ private void initializeResources() {
     }
     chatId = getIntent().getIntExtra(CHAT_ID_EXTRA, -1);
     if(chatId == DcChat.DC_CHAT_NO_CHAT)
-      throw new IllegalStateException("can't display a conversation for no chat.");
+      chatId = -1;
```

Comment:
mmh, actually this seems to be there to prevent from opening the trash chat that happens to be `chatId == 0`?

Reply:
nope. the "trash" has id 6 == DC_CHAT_ID_TRASH.

and at comparable places, we're also using `getIntExtra(CHAT_ID_EXTRA, DC_CHAT_NO_CHAT)` 

still, not sure under which circumstances this "throw" can happen and what the gist of the check is

Reply:
> and at comparable places, we're also using getIntExtra(CHAT_ID_EXTRA, DC_CHAT_NO_CHAT)

mmmh maybe it is a bug but we are using a lot `-1` instead of DC_CHAT_NO_CHAT, so in this line code section, if `CHAT_ID_EXTRA` was not present, it would have been set to `-1` and not throwing the exception then

Reply:
yip, i was just mentioning, it is not an error to deal with -1 as well as 0.

not crashing is probably not that bad, but we could also do `finish(); return` - however, for that change, it may be good to have it reproducible (not that we're called when the activity is already in finishing or so ..)


why is the PR closed, btw?